mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Surgex.Mixfile do
   def project do
     [
       app: :surgex,
-      version: "4.14.0",
+      version: "4.14.1",
```

Comment:
Not sure if I should bump it to `4.14.1` or to `4.15.0`

Reply:
Minor bugfix, a patch version bump is ok.